UPGRADE-1.x.md

```diff
@@ -1,2 +1,10 @@
 UPGRADE 1.x
 ===========
+
+UPGRADE FROM 1.x to 1.x
```

Comment:
We should try to remember this while releasing. Would be great to have a keyword and have the release tool warn us that we need to do something.